packages-exp/messaging-exp/src/api.ts

```diff
@@ -49,13 +49,13 @@ export function getMessaging(app: FirebaseApp = getApp()): FirebaseMessaging {
 }
 
 /**
- * Subscribes the messaging instance to push notifications. Returns an FCM registration token
+ * Subscribes the Messaging instance to push notifications. Returns an FCM registration token
  * that can be used to send push messages to that messaging instance.
  *
  * If a notification permission isn't already granted, this method asks the user for permission.
  * The returned promise rejects if the user does not allow the app to show notifications.
  *
- * @param messaging - the messaging instance.
+ * @param messaging - the Messaging instance.
```

Comment:
Aha -- these are the ones that look odd in the tables because "The" should be capitalized.

It's a tiny nit, but in the table lists, these look like sentences, and an uncapitalized sentence is a bad look.

Can I give you this as a global comment, to apply to all parameter descriptions?  Thanks!